features/admin.applications.v1/components/settings/sign-on-methods/components/sign-on-methods-core.tsx

```diff
@@ -899,6 +899,59 @@ export const SignOnMethodsCore: FunctionComponent<SignOnMethodsCorePropsInterfac
         );
     };
 
+    const renderLegacyAuthenticationFlowBuilder = (): ReactElement => {
+        if (isLoading || isAuthenticatorsFetchRequestLoading) {
+            return (
+                <ContentLoader inline="centered" active />
+            );
+        }
+
+        const isDefaultAuthFlow: boolean = isDefaultFlowConfiguration();
```

Comment:
Is it worth storing the return value in memory if we are not reusing the variable?